docs/building.md

```diff
@@ -5,18 +5,20 @@ Using the configuration provided by this repository requires a build of OpenTele
 * [the `hostmetrics` receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver#readme): Lives in the contrib repository
 * [the `resourcedetection` processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor): Lives in the contrib repository
 * [the `metricstransform` processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor#readme): Lives in the contrib repository
+* [the `healthcheck` extension](ihttps://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/healthcheckextension#readme): Lives in the contrib repository
 * [the `timestamp` processor](../timestampprocessor): Lives in this repository
```

Comment:
Is this necessary anymore? As we're doing this work in shepherd now.

Reply:
I'm torn.. I know some teams are using it with `round_to_nearest` set to 10s, 15s, or 30s and I think we're doing 1s 'bucketing' on ingest now?

Reply:
That's correct. If folks want to up the number, this is still valuable! Wasn't aware that was an active use-case :)